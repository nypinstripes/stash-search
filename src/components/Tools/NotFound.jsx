import { withRouter } from 'react-router-dom';
import React from 'react';
import SvgIcon from './SvgIcon';
import SvgSymbol from './SvgSymbol';

const NotFound = props => (
  <div className="not-found">
    <h1>Zoink!</h1>
    <div className="not-found-icon">
      <div className="robot-body">
        <SvgSymbol symbolId="#robot-body" />
      </div>
      <div className="robot-face">
        <SvgIcon name="robot" />
      </div>
    </div>
    <p>Didn&apos;t find that one.</p>
  </div>
);

export default withRouter(NotFound);
