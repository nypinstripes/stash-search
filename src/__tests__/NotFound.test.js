import React from 'react';
import { create } from 'react-test-renderer';
import NotFound from '../components/Tools/NotFound';

test('snapshot', () => {
  const notFoundComponent = create(<NotFound />);

  expect(notFoundComponent.toJSON()).toMatchSnapshot();
});
