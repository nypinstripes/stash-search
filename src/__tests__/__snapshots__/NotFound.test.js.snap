// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="not-found"
>
  <h1>
    Zoink!
  </h1>
  <div
    className="not-found-icon"
  >
    <div
      className="robot-body"
    >
      <svg
        height="0"
        width="0"
      >
        <use
          x="0"
          xlinkHref="#robot-body"
          y="0"
        />
      </svg>
    </div>
    <div
      className="robot-face"
    >
      <svg
        className="svg-icon-container"
        height="100%"
        viewBox="0 0 100 100"
        width="100%"
      >
        <path
          d="M40.5801239 49.4435959l2.8297653-2.8297691-5.589489-5.5894852 5.589489-5.5894852-2.8297653-2.8297653-5.5894852 5.5894852-5.589487-5.5894852-2.8297673 2.8297653 5.589489 5.5894852-5.589489 5.5894852 2.8297673 2.8297691 5.589487-5.5894852z"
          id="robot-eye-left"
        />
        <path
          d="M59.4198761 49.4435959l5.5894852-5.5894852 5.5894851 5.5894852 2.829773-2.8297691-5.5894852-5.5894852 5.5894852-5.5894852-2.829773-2.8297653-5.5894851 5.5894852-5.5894852-5.5894852-2.8297653 2.8297653 5.5894851 5.5894852-5.5894851 5.5894852z"
          id="robot-eye-right"
        />
        <path
          d="M62.0074883 68.0411987c1.1066933 0 2.0012474-.8945618 2.0012474-2.0012512 0-7.7228203-6.2859154-14.0087395-14.0087357-14.0087395s-14.0087395 6.2859192-14.0087395 14.0087395c0 1.1066895.8945618 2.0012512 2.0012474 2.0012512h24.0149804zM50 56.0337067c4.8330154 0 8.8755379 3.444149 9.8041191 8.0049896H40.1938858C41.1244659 59.4778557 45.1669846 56.0337067 50 56.0337067z"
          id="robot-mouth"
        />
      </svg>
    </div>
  </div>
  <p>
    Didn't find that one.
  </p>
</div>
`;
